scripts/swagger.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-mkdir gen gen/models gen/restapi
-swagger generate server -f ./swagger.yaml -m gen/models -s gen/restapi -A ApnaSchool
+if [ ! -d "gen" ]; then
```

Comment:
Do we still need this after command change?

Reply:
yes we do

Reply:
Yes we need it in case gen folder is not there else it will make `restful` folder in main directory